README.md

```diff
@@ -62,42 +71,81 @@ We will also keep track of the actions that actually took place and readings tak
 | name       | `String`      |
 | sensor_id  | `Foreign Key` |
 | value      | `Float`       |
-| created_at | `Datetime`   |
+| created_at | `Datetime`    |
 
 ### Scheduled Actions
 
 | Name       | Type          |
 | ---------- | ------------- |
 | id         | `Key`         |
 | name       | `String`      |
-| actuator  | `Foreign Key` |
-| data       | `String`         |
-| created_at | `Datetime`   |
+| actuator   | `Foreign Key` |
+| data       | `String`      |
+| created_at | `Datetime`    |
 
 ### Reactive Actions
 
 | Name       | Type          |
 | ---------- | ------------- |
 | id         | `Key`         |
 | name       | `String`      |
-| actuator  | `Foreign Key` |
-| data       | `String`         |
-| created_at | `Datetime`   |
+| actuator   | `Foreign Key` |
+| data       | `String`      |
+| created_at | `Datetime`    |
 
 # Config
 
 The Configurations Table will be used in conjunction with logging data in [Mother Nature](https://github.com/Olin-Hydro/mother-nature)
 
 ### Operational Config File
 
-| Name                   | Type            |
-| ---------------------- | --------------- |
-| id                     | `Key`           |
-| garden_id              | `Foreign Key`   |
-| sensor_ids             | `Json` |
-| scheduled_actuator_ids | `Json` |
-| reactive_actuator_ids  | `Json` |
-| created_at             | `Datetime`     |
+| Name                | Type           |
+| ------------------- | -------------- |
+| id                  | `Key`          |
+| garden_id           | `Foreign Key`  |
+| name                | `String`       |
+| sensors             | `[SS]` |
+| scheduled_actuators | `[SAS]`     |
+| reactive_actuators  | `[RAS]`     |
+| created_at          | `Datetime`     |
+
```

Comment:
Just had another thought, we need to add routes for retrieving the garden configurations as well. I think we will want to search for them by ID and by Name (for ease of readability and reference in mother nature)

Reply:
Alternatively, we could also remove gardenID from the configuration. Then give gardens a configuration field that we can embed this information into. I think that might make more sense overall, keeps the garden tied to a config and is more clear what config the garden is currently using